overlay_new.py

```diff
@@ -32,10 +32,22 @@ def data_field_coord(x, y):
             """Coordinates of the data field in column x, row y."""
             return col_coords[x], row_coords[y]
 
+        # Dimensions of the left TransparentRectangle
+        left_rect = [
+            (0, row_coords[0] - DataField.height),
```

Comment:
Can you add a bit of padding to the top edge? Since the last review the rectangles touch the top of the text.
![image](https://user-images.githubusercontent.com/17876556/97096474-91275680-16b8-11eb-9efe-8cd9b9aa45d6.png)

```suggestion
            (0, row_coords[0] - DataField.height - spacing),
```